tests/utils/mock_server.py

```diff
@@ -1363,8 +1387,7 @@ def graphql():
                 return json.dumps({"data": {"project": {"runQueues": []}}})
 
         if "query GetRunQueueItem" in body["query"]:
-            ctx["run_queue_item_check_count"] += 1
-            if ctx["run_queue_item_check_count"] > 1:
```

Comment:
do you need to remove this ctx?

Reply:
Yeah, I've modified the associated test and made a more clear context to indicate how the mock server should respond to the query based on context